### PR TITLE
add insync

### PR DESCRIPTION
**Pull request by masakichi**
https://www.insynchq.com/blog/google-drive-client-for-linux/

### CODE REVIEW

File: insync/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Contributor: Zhengyu Xu <xzy3186@gmail.com>
+
+pkgname=insync
+pkgver=1.5.5
+_pkgver=37367
+_dist=artful
+pkgrel=1
+pkgdesc="An unofficial Google Drive client that runs on Linux, with support for various desktops"
+url="https://www.insynchq.com/downloads"
+license=('custom:insync')
```
Comment (yan12125) [inline insync/PKGBUILD]: Does its license allow binary redistribution?

Comment (masakichi) [reply insync/PKGBUILD]: I don't find license information on its home page. 

well, this software itself distribute a `.deb` file, I think it's not a big problem?  Just like enpass-bin that already in archlinuxcn repo.

**Comment by zsrkmyn** [comment]
> You may **not** use, copy, adapt, modify, prepare derivative works based upon, **distribute**, license, sell, transfer, publicly display, publicly perform, transmit, stream, broadcast or otherwise exploit the Services or Content (other than the Content you have provided), except as expressly permitted in these Terms.

https://www.insynchq.com/terms

看起来不行？

**Comment by masakichi** [comment]
好的，一会儿我关了这个 PR

2018年12月14日(金) 午後7:39、Stephen さん（notifications@github.com）のメッセージ:

**Pull request closed without merging**